client/assets/images/_icons_test-run-card/icon_reviewed.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" fill="none" viewBox="0 0 20 20">
```

Comment:
Usually, .svg files do not need width/height attrs if they have viewBox

Reply:
fixed